### PR TITLE
Add register route

### PR DESCRIPTION
**Pull request by Porquepix**

### CODE REVIEW

File: .env.dev [hunk 1]
```diff
@@ -18,4 +18,7 @@ export TYPEORM_ENTITIES_DIR="src/entities"
 export TYPEORM_MIGRATIONS_DIR="src/migrations"
 
 export DATABASE_TYPE="postgres"
+export DATABASE_SSL=false
 export DATABASE_URL="postgres://postgres:root@localhost:5434/dev_prello"
+
+export TEST_DATABASE_URL="postgres://postgres:root@localhost:5434/test_prello"
```
Comment (YvesLongchamp) [inline .env.dev:24]: Is there any password in this ?

File: src/app.ts [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+export const app = express()
+
+app.set('port', config.server.port)
+app.use(compression())
+app.use(cors())
+app.use(bodyParser.json())
+app.use(bodyParser.urlencoded({ extended: true }))
+/*
+app.use('*', (req, res, next) => {
```
Comment (YvesLongchamp) [inline src/app.ts:26]: Is this necessary, all the commented part ?

File: src/bl/notificationFacade.ts [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
+    //     if (notifications) {
+    //         return notifications
+    //     } else {
+    //         throw new NotFoundException('No Notification was found')
+    //     }
+    // }
 
     static async getById(notificationId: number): Promise<Notification> {
-        const notification = await getEntityManager()
+        const notification = await getManager()
```
Comment (YvesLongchamp) [inline src/bl/notificationFacade.ts:23]: What is the difference, between entityManager and manager ?

**Review by YvesLongchamp — COMMENTED** [review]
There are a few points that need to be clarified.

File: .env.dev (hunk 1 above)
Comment (Porquepix) [reply .env.dev:24]: No only local db password.

File: src/app.ts (hunk 2 above)
Comment (Porquepix) [reply src/app.ts:26]: Yes, because this code is down. It will be fixed in my next PR.

File: src/bl/notificationFacade.ts (hunk 3 above)
Comment (Porquepix) [reply src/bl/notificationFacade.ts:23]: EntityManager is the old version of Manager in TypeORM.

**Review by YvesLongchamp — APPROVED** [review]
Now that everything is clear, it is ok for me :)

**Pull request merged**